src/ResourceManager/DataLakeStore/Commands.DataLakeStore.Test/Commands.DataLakeStore.Test.csproj

```diff
@@ -90,6 +88,9 @@
     <None Include="SessionRecords\Microsoft.Azure.Commands.DataLakeStore.Test.ScenarioTests.AdlsAliasTests\TestAdlsFirewallRules.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Include="SessionRecords\Microsoft.Azure.Commands.DataLakeStore.Test.ScenarioTests.AdlsAliasTests\TestAdlsVirtualNetworkRules.json">
```

Comment:
@lewu-msft the build is currently failing because the `.json` files you added a reference to in this file weren't added in the PR